functions/ramp.h

```diff
@@ -1,9 +1,9 @@
 #ifndef FUNCTIONS_RAMP_H
 #define FUNCTIONS_RAMP_H
 
+// Usage: Remap<RampF,COLOR>
```

Comment:
No, it should be:
Usage: RampF

Also, the description should be updated to:
Returns 0 at base and 32768 at tip.
Example: Mix<RampF, COLOR1, COLOR2, COLOR3>
This would do the same thing as Gradient<COLOR1, COLOR2, COLOR3>

Reply:
done.